lagunita/lms/static/sass/multicourse/_courses.scss

```diff
@@ -0,0 +1,45 @@
+@import 'lms/static/sass/base/grid-settings';
+@import 'neat/neat';
+
+.find-courses {
```

Comment:
What is going on here?
Which issue does this address?
How does this differ from how things are displayed upstream?

Reply:
I'm having to redefine the [platform css rules](https://github.com/Stanford-Online/edx-platform/blob/master/lms/static/sass/multicourse/_courses.scss#L93) because in our case course discovery is still enabled, but since we are hiding the sidebar we don't want it to leave an extra column for the sidebar